packages/react-divider/src/Divider.stories.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
-import { ClockIcon } from '@fluentui/react-icons-mdl2';
-import { Divider, DividerProps } from '@fluentui/react-divider';
+import { ClockIcon } from './tmp-icons.stories';
+import { Divider, DividerProps } from './index';
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
 export interface DividerStoryProps {
```

Comment:
```suggestion
interface DividerStoryProps {
  label?: string;
  children?: React.ReactChild | React.ReactChild[];
  className?: string;
}
```

- we can do better for children typings instead of turning lint off 💪